.github/workflows/githubactions.yml

```diff
@@ -0,0 +1,61 @@
+# -----------------------------------------------------------------------------
+# Copyright Siemens AG, 2021.
+# Part of the SW360 Portal Project.
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# -----------------------------------------------------------------------------
+
+name: SW360
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  COUCHDB_USER: admin
+  COUCHDB_PASSWORD: password
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Setup CouchDB
+      uses: iamssen/couchdb-github-action@master
+      with:
+          couchdb-version: '3.1'
+
+    - name: Update properties with DB credentials
+      run: |
+        sudo apt-get update -qq
+        sudo sed -i 's/^couchdb.user\s*=/& admin/' build-configuration/test-resources/couchdb-test.properties
+        sudo sed -i 's/^couchdb.password\s*=/& password/' build-configuration/test-resources/couchdb-test.properties
+        sudo sed -i -e '$acouchdb.username=admin' libraries/lib-datahandler/src/test/resources/databasetest.properties
+        sudo sed -i -e '$acouchdb.password=password' libraries/lib-datahandler/src/test/resources/databasetest.properties
+
+    - name: Verify license headers
+      run: |
+        chmod +x .github/testForLicenseHeaders.sh
+        bash .github/testForLicenseHeaders.sh
```

Comment:
The Step is not running and causing failure, however the build is successful. It should fail in case of any failure

![image](https://user-images.githubusercontent.com/56534710/125437076-6f81d8ee-1454-4b2e-8250-095446e07060.png)

Reply:
Some are suggesting this type of error is normal in multi-CPU system. 

https://superuser.com/questions/554855/how-can-i-fix-a-broken-pipe-error

Not sure how we can fix it. Though the the script is doing it's job. Github Actions runner Ref: https://github.com/eclipse/sw360/actions/runs/1029154614